synapse/tests/common.py

```diff
@@ -38,6 +40,8 @@ class TooFewEvents(Exception): pass
 
 testdir = os.path.dirname(__file__)
 
+writeCerts(testdir)
```

Comment:
Do we need to call this? Can't we just put them in here?

Reply:
Yup - moving the files from `synapse.tests` ->  `synapse.data` means we need to do a check and drop if we want existing tests to work. otherwise we'd have to touch a bunch of tests.  the files are gitignored so they won't muck up a dev environment.

Reply:
ok